split_expenses_ui/src/pages/QuickExpense/QuickExpense.jsx

```diff
@@ -293,13 +309,13 @@ class QuickExpense extends Component {
               </div>
             </section>
           </div>
-          <ProcessExpense
+           {(!isLoading && products.length > 0) ?<ProcessExpense
```

Comment:
Shall we validation if the users are present or not before submitting the request? we can save a backend network call and request the client to add the users. Can we add this additional condition before displaying the quick expense table?

Reply:
Thanks. Added a check for **users** as well.